deploy/helm/prometheus-overrides.yaml

```diff
@@ -228,13 +228,13 @@ prometheus:
         sourceLabels: [__name__]
     - # control plane metrics
       # coredns
-      url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane
+      url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane.coredns
       writeRelabelConfigs:
       - action: keep
         regex: coredns
         sourceLabels: [job]
     - # etcd server
-      url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane
+      url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane.kube-etcd
```

Comment:
Fixes prometheus issue. It has been saying that configuration for `http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.control-plane` is duplicated

Reply:
The only thing i find misleading with this setup is that we collect from other control plane components and do not prefix them with control-plane.  Any reason to just remove the control-plane part?

Reply:
Yes, the fluentd configuration will be more complex: either more complex regex for tags or output section per endpoint

Reply:
How many metrics is each of these producing? We need to be conscious of blacklisting and if we are close to the limit, we should distribute across multiiple HTTP sources to prevent that.